### ISSUE TITLE
Build is broken in latest `nixos-unstable`

### ISSUE THREAD

**Issue by panchoh** [nilp0inter/autofirma-nix#6]
```
error: builder for '/nix/store/11q0rs3y40zqbjb3zmy4qqbgq97mz8f5-autofirma-1.8.2.drv' failed with exit code 1;
       last 10 log lines:
       > [ERROR] Plugin org.apache.maven.plugins:maven-install-plugin:3.1.2 or one of its dependencies could not be resolved:
       > [ERROR]  Cannot access central (https://repo.maven.apache.org/maven2) in offline mode and the artifact org.apache.maven.plugins:maven-install-plugin:jar:3.1.2 has not been downloaded from it before.
       > [ERROR] -> [Help 1]
       > [ERROR]
       > [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
       > [ERROR] Re-run Maven using the -X switch to enable full debug logging.
       > [ERROR]
       > [ERROR] For more information about the errors and possible solutions, please read the following articles:
       > [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginResolutionException
       > 
       For full logs, run 'nix log /nix/store/11q0rs3y40zqbjb3zmy4qqbgq97mz8f5-autofirma-1.8.2.drv'.
```

**Comment by panchoh** [comment]
The relevant info of the full logs:
```
[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Build Order:
[INFO] 
[INFO] jmulticard-base                                                    [pom]
[INFO] jmulticard                                                         [jar]
[INFO] jmulticard-jsr268                                                  [jar]
[INFO] jmulticard-ui                                                      [jar]
[INFO] jmulticard-jse                                                     [jar]
[INFO] 
[INFO] --------------< es.gob.afirma.jmulticard:jmulticard-base >--------------
[INFO] Building jmulticard-base 1.8                                       [1/5]
[INFO]   from pom.xml
[INFO] --------------------------------[ pom ]---------------------------------
[WARNING] The POM for org.apache.maven.plugins:maven-install-plugin:jar:3.1.2 is missing, no dependency information available
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for jmulticard-base 1.8:
[INFO] 
[INFO] jmulticard-base .................................... FAILURE [  0.032 s]
[INFO] jmulticard ......................................... SKIPPED
[INFO] jmulticard-jsr268 .................................. SKIPPED
[INFO] jmulticard-ui ...................................... SKIPPED
[INFO] jmulticard-jse ..................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.124 s
[INFO] Finished at: 2024-07-09T04:57:34Z
[INFO] ------------------------------------------------------------------------
[ERROR] Plugin org.apache.maven.plugins:maven-install-plugin:3.1.2 or one of its dependencies could not be resolved:
[ERROR]         Cannot access central (https://repo.maven.apache.org/maven2) in offline mode and the artifact org.apache.maven.plugins:maven-install-plugin:jar:3.1.2 has not been downloaded from it before.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginResolutionException
```

**Comment by nilp0inter** [comment]
Somehow fixed in 2366e6a60f3372422d393ebe6ba6f0af285cb1c2 🤷
